testssl.sh

```diff
@@ -10349,8 +10352,8 @@ fd_socket() {
                acap|acaps) # ACAP = Application Configuration Access Protocol, see https://tools.ietf.org/html/rfc2595
                     fatal "ACAP Easteregg: not implemented -- probably never will" $ERR_NOSUPPORT
                     ;;
-               xmpp|xmpps) # XMPP, see https://tools.ietf.org/html/rfc6120
-                    starttls_xmpp_dialog
+               xmpp|xmpps|xmpp-server) # XMPP, see https://tools.ietf.org/html/rfc6120
+                    starttls_xmpp_dialog "$STARTTLS_PROTOCOL"
```

Comment:
Minor thing: ``starttls_xmpp_dialog()`` isn't requesting an argument and even the global ``$STARTTLS_PROTOCOL`` isn't needed.